<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LR1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="coursesComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="courseGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>911</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Курс: &quot;&quot;</string>
    </property>
    <widget class="QGroupBox" name="studentsGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Студенты</string>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>211</width>
        <height>431</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>209</width>
         <height>429</height>
        </rect>
       </property>
       <widget class="QListWidget" name="studentsListWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>201</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="controlsGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>221</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Элементы контроля</string>
     </property>
     <widget class="QListWidget" name="controlElementsListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="addControlElementButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить элемент контроля</string>
     </property>
    </widget>
    <widget class="QLabel" name="formulaLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Формула оценки:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addStudentButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить студента</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addCoursePushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить курс</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
